k256/src/arithmetic/scalar/scalar_8x32.rs

```diff
@@ -528,6 +533,12 @@ impl From<u32> for Scalar8x32 {
     }
 }
 
+impl From<u64> for Scalar8x32 {
+    fn from(k: u64) -> Self {
+        Self([(k & 0xFFFF) as u32, (k >> 32) as u32, 0, 0, 0, 0, 0, 0])
```

Comment:
I think I did this right, but someone please double check me (or maybe I should write a test)